app/src/main/java/com/limelight/utils/ShortcutHelper.java

```diff
@@ -127,10 +128,42 @@ public void createAppViewShortcut(String id, ComputerDetails details, boolean fo
         createAppViewShortcut(id, details.name, details.uuid.toString(), forceAdd);
     }
 
+    @TargetApi(Build.VERSION_CODES.O)
+    public boolean createPinnedGameShortcut(String id, Bitmap iconBits, String computerName, String computerUuid, String appName, String appId) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && sm.isRequestPinShortcutSupported()) {
```

Comment:
`@TargetApi(Build.VERSION_CODES.O)` already implies `Build.VERSION.SDK_INT >= Build.VERSION_CODES.O`. That annotation means callers must not use this on < O. This check should be removed or hoisted into the caller if it's truly needed.